packaging/packager_rpm.sh

```diff
@@ -20,7 +20,7 @@ dnf -y install tesseract tesseract-devel
 dnf -y install gcc-c++
 pip3 install pytesseract==0.3.4 tesserocr==2.5.1
 # deep learning
-pip3 install torch==1.8.1 torchvision==0.9.1
+pip3 install torch==1.13.0 torchvision==0.14.0
```

Comment:
I am confused - some torch versions are 1.13.0 and others are 1.13.1.

Reply:
Corrected the versions.